packages/itwinui-react/src/core/ColorPicker/ColorInputPanel.tsx

```diff
@@ -23,6 +30,22 @@ type ColorInputPanelProps = {
    * @default ['hsl', 'rgb', 'hex']
    */
   allowedColorFormats?: ('hsl' | 'rgb' | 'hex')[];
+  /**
+   * Passes props to the color picker section label.
+   */
+  panelLabelProps?: React.ComponentProps<'div'>;
+  /**
+   * Passes props to the color input container.
+   */
+  colorInputContainerProps?: React.ComponentProps<'div'>;
```

Comment:
If you've renamed any props, make sure changeset is up-to-date!

Reply:
My bad for missing this. I have edited the PR description and changeset.